reckon-gradle/src/main/java/org/ajoberstar/reckon/gradle/ReckonExtension.java

```diff
@@ -58,6 +59,7 @@ public void setPreRelease(PreReleaseStrategy preRelease) {
   }
 
   public VcsInventorySupplier git(Grgit grgit) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public method by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=squid%3AUndocumentedApi)